Entities/Actor.h

```diff
@@ -1199,26 +1199,64 @@ ClassInfoGetters
 
 
 	/// <summary>
-	/// Returns the defined sound to be played on death.
+	/// Gets this Actor's Death Sound. Ownership is not transferred!
 	/// </summary>
-	/// <returns>A sound with the death sample of this actor.</returns>
-	SoundContainer GetDeathSound() const { return m_DeathSound; }
+	/// <returns>The SoundContainer for this Actor's Death Sound.</returns>
+	SoundContainer* GetDeathSound() const { return m_DeathSound; }
 
+	/// <summary>
+	/// Sets this Actor's Death Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this Actor's Death Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.